queries/aws_queries/AWS_Authentication_from_CrowdStrike_Unmanaged_Device_FDREvent.yml

```diff
@@ -0,0 +1,22 @@
+# This file is the part of the Crowdstrike FDREvent migration, and it's the equivalent of
+# https://github.com/panther-labs/panther-analysis/blob/b61db1ecf3967c5f6a44c1782f8891fd5f54384d/queries/aws_queries/AWS_Authentication_from_CrowdStrike_Unmanaged_Device.yml
+#
+AnalysisType: scheduled_query
+Description: Detects AWS Authentication events with IP Addresses not found in CrowdStrike's AIP List
+Enabled: false
+Query: |
+    SELECT *
+    FROM aws_cloudtrail
+    WHERE p_occurs_since('1 days')
+      AND eventName IN ('ConsoleLogin', 'SignIn', 'GetSessionToken')
+      AND eventSource IN ('sts.amazonaws.com', 'signin.amazonaws.com')
+      AND sourceIPAddress NOT IN
+        (
+          SELECT DISTINCT aip
+          FROM crowdstrike_fdrevent
```

Comment:
Given the adoption of Snowflake as a datalake backend qualifying with database/schema would work with the benefit that you mention. Otherwise the queries become incompatible with Athena.